Cargo.toml

```
@@ -39,22 +39,22 @@ thiserror = { version = "1.0.40" }
 wat = { version = "1.0.71" }
 
 # Substrate dependencies
-frame-metadata = { version = "16.0.0" }
-frame-support = { version = "36.0.0" }
-frame-system = { version = "36.1.0" }
-pallet-assets = { version = "37.0.0" }
-pallet-balances = { version = "37.0.0" }
-pallet-contracts = { version = "35.0.0" }
-pallet-timestamp = { version = "35.0.0" }
-sp-core = { version = "34.0.0" }
-sp-externalities = { version = "0.29.0" }
-sp-io = { version = "37.0.0" }
-sp-runtime-interface = { version = "28.0.0", features = ["std"] }
+frame-metadata = { version = "20.0.0" }
+frame-support = { version = "40.1.0" }
+frame-system = { version = "40.1.0" }
+pallet-assets = { version = "42.0.0" }
+pallet-balances = { version = "41.1.0" }
+pallet-contracts = { version = "40.1.0" }
+pallet-timestamp = { version = "39.0.0" }
+sp-core = { version = "36.1.0" }
+sp-externalities = { version = "0.30.0" }
+sp-io = { version = "40.0.1" }
+sp-runtime-interface = { version = "29.0.1", features = ["std"] }
 
 # Local
 drink = { path = "crates/drink/drink" }
 ink_sandbox = { path = "crates/ink-sandbox" }
-pop-api = { git = "https://github.com/r0gue-io/pop-node.git", branch = "main" }
+pop-api = { git = "https://github.com/r0gue-io/pop-node.git", branch = "chungquantin/sync-pallet-nfts-stable2503" }
```

```suggestion
pop-api = { git = "https://github.com/r0gue-io/pop-node.git", branch = "main" }
```

Needs to be updated after https://github.com/r0gue-io/pop-node/pull/558 merged to pop-node `main`